paddle/infrt/dialect/mlir_loader.cc

```diff
@@ -36,7 +36,7 @@ mlir::OwningModuleRef LoadMlirSource(mlir::MLIRContext* context,
                                      const std::string& mlir_source) {
   // context->allowUnregisteredDialects();
   RegisterCinnDialects(context->getDialectRegistry());
-  context->getDialectRegistry().insert<mlir::StandardOpsDialect>();
+  // context->getDialectRegistry().insert<mlir::StandardOpsDialect>();
```

Comment:
Always comment with the reason of disabling the exsting code for review.

Reply:
Done